sbat.c

```diff
@@ -4,27 +4,28 @@
  */
 
 #include "sbat.h"
+#include <string.h>
 
 CHAR8 *
-get_sbat_field(CHAR8 *current, CHAR8 *end, const CHAR8 ** field, char delim)
+get_sbat_field(CHAR8 *current, CHAR8 *end, const CHAR8 **field, char delim)
```

Comment:
I see what happened. You also squashed @vathpela's patch that was cleaning up the formatting issues. That should be left as a separate patch and I think that can already be merged.